sample/F8/tracks

```diff
@@ -0,0 +1,209 @@
+[{
+		"id": 1,
+		"name": "ADS & MONETIZATION",
```

Comment:
See in raw format, it should be visible

Easiest thing will be to copy the quotation mark from `"name"` and paste it around `ADS & MONETIZATION` and `C`